app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -37,7 +37,11 @@ def call
            end
           end
 
-          maybe_redirect_to_verify_info if updating_state_id?
+          if !pii_from_user.key?(:address1)
+            redirect_to_address
+          elsif (updating_state_id?)
+            maybe_redirect_to_verify_info
+          end
```

Comment:
I added this additional evaluation in else I also go to /verify_info.  I also added redirect_to_address inside temp_maybe_redirect_to_verify_info_helper.  Naming convention tells me this is not the best place for it but I also think this will only be temp.  (If I comment out `maybe_redirect_to_verify_info if updating_state_id?` without my code- it takes me where I'd expect (address or verify) so I believe when verify_info is entirely in place `rediect_to_address` can also be deleted.